source/community/reactnative/src/components/actions/select-Action.js

```diff
@@ -55,6 +55,10 @@ export class SelectAction extends React.Component {
 			return null;
 		}
 
+		this.payload = this.props.selectActionData;
+		this.onExecuteAction = undefined;
+		this.toggleVisibilityForElementWithID = undefined;
```

Comment:
Should this be called on every render? could you please explain the need for this @frags51

Reply:
Yes, it needs to be. This change is similar to the changes we made earlier.
OnExecute action and togglevisibility are set to the correct value in case of every render. So this should not affect correctness. We need to change the payload on every render because props must be source of truth. Since it is just variable assignment, it is not a costly operation.